lib/loga/sidekiq/job_logger.rb

```diff
@@ -0,0 +1,54 @@
+module Loga
+  module Sidekiq
+    class JobLogger
+      include Loga::Utilities
+
+      EVENT_TYPE = 'sidekiq'.freeze
+
+      attr_reader :started_at, :data
+
+      def initialize
+        @started_at = Time.now
+        @data = {}
+      end
+
+      def call(item, _queue)
+        yield
+        data['status'] = 'success'
+      rescue Exception # rubocop:disable Lint/RescueException
```

Comment:
We should pass the exception to the `Loga::Event` too so that is can be viewed in the logs. The exception should also influence the log level, `info` for normal execution, `fatal` when an exception occurs.